test/CMakeLists.txt

```diff
@@ -120,8 +120,9 @@ endfunction()
 
 
 # Include generated files (moc ouput, build configuration)
-include_directories("${CMAKE_CURRENT_BINARY_DIR}/../src")
-include_directories("${CMAKE_CURRENT_BINARY_DIR}")
+include_directories("${PROJECT_BINARY_DIR}/src")
+include_directories("${PROJECT_BINARY_DIR}")
```

Comment:
No, this remains CMAKE_CURRENT_BINARY_DIR (for files generate in test subdir of binary tree).